config/rbac/role.yaml

```diff
@@ -18,7 +18,7 @@ rules:
   - update
   - watch
 - apiGroups:
-  - operator.openshift.io
+  - externaldns.olm.openshift.io
   resources:
   - externaldns
```

Comment:
I faced this problem during the test of OLM bundle: I changed CRD name to `externaldnses.externaldns.olm.openshift.io` to use the future name.
The problem is that the resources is `externaldnsES` (as specified in CRD).